src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -3,39 +3,74 @@ title: Usage
+The button with the strongest visual weight will attract the most attention.
+
+| Button name                 | Visual style                                                                                            | Usage                                                                                                                                                                                               |
+| --------------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Call-to-action (CTA) button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png)       | funnel users into the most important action for your business ex: "Apply for funding" or "Watch live streaming". Limit to one CTA on a given page, the same CTA can be repeated within in the page. |
+| Primary button              | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_Default.png)   | Only 1 primary action per context/section of the page, Primary buttons can appear in a few occurence on a page. ex: "submit","register"...                                                          |
+| Secondary button            | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Secondary_Default.png) | Secondary action that users can perform on the page and it can be used multiple times per context/section in the page. ex: "show more", "expand", "see details".                                    |
+| Ghost button                | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Ghost_Default.png)     | When an action does not require dominance on a page. Often use next to a primary action ex: On a form context/section "Submit" as primary, "Cancel" as a ghost.                                     |
+| Search button               | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Search_Default.png)    | use exclusively for the search function.                                                                                                                                                            |
 
 ## When not to use
 
-Do not use Buttons to send the user to another page or for navigational purposes. Use links instead, as they are not associated with an action.
+For navigational purposes. If the action triggered sends a user to a different page, opens up a page in a new tab or jump to another section within the current page, use <Link to="/ec/components/navigation/link/usage/">links</Link> instead because these actions don’t change anything on the website’s state.
+The user is passive, browsing through content (read only).
```

Comment:
```suggestion
Should not use too many buttons on a page.
```

Reply:
This removes an important point.
The suggested replacement is more a "Don't" than a "when not to use"